src/pages/Loading.tsx

```diff
@@ -0,0 +1,12 @@
+
+import React from 'react';
+
+const Loading = () => {
+  return (
+    <div className="flex items-center justify-center h-screen">
+        <div className="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-gray-900"></div>
```

Comment:
change `border-gray-900 ` to `border-base-darkest` to work with the color theming.